modules/xmpp/strophe.jingle.js

```diff
@@ -331,7 +331,7 @@ export default class JingleConnectionPlugin extends ConnectionPlugin {
                 `Marking session from ${fromJid
                 } as ${isP2P ? '' : '*not*'} P2P`);
 
-            const iceConfig = isP2P ? this.p2pIceConfig : this.jvbIceConfig;
+            const pcConfig = isP2P ? this.p2pIceConfig : this.jvbIceConfig;
```

Comment:
Do we not want to rename `this.p2pIceConfig` and `this.jvbIceConfig` ? Me thinks `this.pcConfig.p2p` and `this.pcConfig.jvb` looks moar better :-P

Reply:
Actually, `this.p2pIceConfig `and `this.jvbIceConfig` have the ice servers info, that is why I didn't change them.

Reply:
Oh, nevermind then!